lib/lambda_ethereum_consensus/beacon/pending_blocks.ex

```diff
@@ -31,9 +31,9 @@ defmodule LambdaEthereumConsensus.Beacon.PendingBlocks do
     GenServer.cast(__MODULE__, {:add_block, signed_block})
   end
 
-  @spec is_pending_block(Types.root()) :: boolean()
```

Comment:
Im removing this on https://github.com/lambdaclass/lambda_ethereum_consensus/pull/709